### DISCUSSION TITLE
cloudbuild module's fileMatch doesn't work as expected

### DISCUSSION THREAD

**Discussion by yosmoc** [renovatebot/renovate#18411] [Request Help] [answered]
Hi,

I try to update the docker image version in cloudbuild .yaml file. Our repository has the cloud build yaml files in `.cloudbuild/`, so I write the fileMatch rules like

```json
{
  "cloudbuild": {
    "fileMatch": ["^.cloudbuild/.*\\.yaml$"]
  }
}
```

And there are two files in the `.cloudbuild` folder in the default branch. `build-and-deploy.yaml` and `cloud-run-preview.yaml`.. However, renovate seems detects only `cloud-run-preview.yaml` file.
![image](https://user-images.githubusercontent.com/9694/196658468-f782b297-7094-4952-b50f-039ba4832ffd.png)

Do I have some mistakes in my configuration?

**Comment by viceice** [comment]
try `^\\.cloudbuild\\/.+\\.yaml$`

**Comment by viceice** [comment, marked as answer]
check the debug logs, maybe there is some syntax error or no dependency in that file.

we show only files with valid dependencies on dashboard.

**Reply by yosmoc** [reply]
Thanks. I found the error in the `build-and-deploy.yaml` file.
